src/charm.py

```diff
@@ -3,11 +3,15 @@
 # Licensed under the GPLv3, see LICENSE file for details.
 
 import logging
+import os
+import yaml
 
+from charmhelpers.core import hookenv
 from ops.charm import CharmBase
 from ops.main import main
 from ops.model import ActiveStatus
 from ops.framework import StoredState
+from ops.model import BlockedStatus
```

Comment:
can we group the ops.model imports together, we have one above

Reply:
done